quark/script/__init__.py

```diff
@@ -435,6 +435,23 @@ def getAllStrings(self) -> List[str]:
         apkinfo = self.quark.apkinfo
         return apkinfo.get_strings()
 
+    def isHardcoded(self, argument: str) -> bool:
+        """
+        Check if the method argument is hardcoded into APK.
```

Comment:
Instead of "method argument", I suggest using the word "argument" because we already use it to refer to the same concept in the Quark Script doc (e.g. [here](https://quark-engine.readthedocs.io/en/latest/quark_script.html#behaviorinstance-isargfrommethod-targetmethod)).